cmake/GetLIBUNWINDVersion.cmake

```diff
@@ -0,0 +1,12 @@
+function(GetLIBUNWINDVersion _LIBUNWIND_VERSION)
```

Comment:
I'd use `LibUnwind` instead of `LIBUNWIND` in function names and cmake files.

Reply:
The library's canonical name is `libunwind`, so I would prefer to keep all letters the same case.

Reply:
It looks inconsistent with other libraries we have:

```
$ ls cmake | grep -i findlib
FindLibEIO.cmake
FindLibEV.cmake
FindLIBUNWIND.cmake
FindLibYAML.cmake
```

Reply:
Fixed: changed all library references to `LibUnwind`.